browserstack.err

```diff
@@ -0,0 +1 @@
+[object Object]
```

Comment:
Is this file important? If not pls remove the file. 
However, if it is then you should remove this -> `[object object]`. It is not user-friendly. It seems you're calling a string function on an object.